gwpy/detector/channel.py

```diff
@@ -413,13 +413,17 @@ def query(cls, name, kerberos=None):
             default behaviour will check for credentials and request username
             and password if none are found (`None`)
 
+        kwargs
+            other keyword arguments are passed directly to
+            :func:`ciecplib.get`
```

Comment:
Extremely minor editorial suggestion:
```suggestion
            :func:`ciecplib.query`
```